test/02-teardown.test.js

```diff
@@ -1,149 +1,149 @@
 'use strict'
+const { isWindows } = require('which-runtime')
 const test = require('brittle')
 const path = require('bare-path')
+const os = require('bare-os')
 const hypercoreid = require('hypercore-id-encoding')
 const Helper = require('./helper')
-const harness = path.join(Helper.localDir, 'test', 'fixtures', 'harness')
+const teardownDir = path.join(Helper.localDir, 'test', 'fixtures', 'teardown')
+const teardownOsKillDir = path.join(Helper.localDir, 'test', 'fixtures', 'teardown-os-kill')
+const teardownExitCodeDir = path.join(Helper.localDir, 'test', 'fixtures', 'teardown-exit-code')
 
-test('teardown', async function ({ is, ok, plan, comment, teardown, timeout }) {
-  timeout(180000)
+test('teardown on pipe end', async function ({ ok, is, plan, comment, teardown, timeout }) {
+  if (isWindows) return
```

Comment:
use `skip` test option like this -> https://github.com/holepunchto/localdrive/blob/3ccf20bf12290164ab5379684cc9d07476109e24/test/entry.js#L28

Reply:
updated